scipy/stats/_stats_py.py

```diff
@@ -7231,13 +7239,23 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     the asymptotic distribution is not chi-square, in which case this test
     is not appropriate.
 
+    For Pearson's chi-squared test, the total observed and expected counts must match
+    for the p-value to accurately reflect the probability of observing such an extreme
+    value of the statistic under the null hypothesis.
+    This function may be used to perform other statistical tests that do not require
+    the total counts to be equal. For instance, to test the null hypothesis that
+    ``f_obs[i]`` is Poisson-distributed with expectation ``f_exp[i]``, set ``ddof=-1``
+    and ``sum_check=False``. This test follows from the fact that a Poisson random
+    variable with mean and variance ``f_exp[i]`` is approximately normal with the
+    same mean and variance; the chi-squared statistic standardizes, squares, and sums
+    the observations; and the sum of ``n`` squared standard normal variables follows
+    the chi-squared distribution with ``n`` degrees of freedom.
```

Comment:
Up to you, but I don't think we actually need the derivation here? (I'm sorry I don't have a reference for the application, but it's one of those things where a good reference would be "any statistician knows this".)

Reply:
Well, yeah, I included the sentence only because we didn't have a reference.

Reply:
Alternatively, you could just say more concisely that

> This test follows from asymptotic Normality of the Poisson distribution.

That's not a reference, but gives keywords that would end people up in the right place?

(As for the t-test: I think the point of a reference in this sort of thing is to provide further documentation on (a) what exactly is being done, and (b) why; I don't think it's important to, say, reference Gosset's original paper.)

Reply:
> As for the t-test:

That is correct.